src/notification-manager.coffee

```diff
@@ -46,8 +56,18 @@ class NotificationManager
   #
   # * `message` A {String} message
   # * `options` (optional) An options {Object} with the following keys:
-  #    * `detail` (optional) A {String} with additional details about the
-  #      notification.
+  #    * `buttons` (optional) An {Array} of {Object} where each {Object} has the
+  #      following options:
+  #      * `className` (optional) {String} additional class name add to the
+  #        button. It will already receive `btn btn-info`.
+  #      * `onDidClick` (optional) {Function} callback to call when the button
+  #        has been clicked. The context will be set to the
+  #        {NotificationElement} instance.
+  #      * `text` {String} inner text for the button
+  #    * `description` (optional) A {String} that will be rendered as Markdown
+  #      to describe the notification.
+  #    * `detail` (optional) A preformatted {String} that will be rendered as
```

Comment:
The "preformatted" is important because the `description` is rendered with `white-space: pre;`, as preformatted text. White-space is preserved, like in a stack trace that is indented with spaces or tabs.

Both `description` and `detail` are rendering the string `"this is an error\n      buddy"` in the following screenshot:

<img width="460" alt="screen shot 2016-06-21 at 7 33 38 am" src="https://cloud.githubusercontent.com/assets/29612/16233227/7ee58c7c-3782-11e6-8358-2c0f395e7c1c.png">

Is there a way to make the "preformatted" more clear?